### ISSUE TITLE
Please change keyboard shortcut for "find" in text editor to command-F on macOS

### ISSUE THREAD

**Issue by Andreas0602** [mucommander/mucommander#423]
On macOS, please change the keyboard shortcut for "find" in the built-in text editor from control-F to command-F, or make it configurable in Settings -> Keyboard Shortcuts.

**Comment by ahadas** [comment]
Duplicate of #334

**Comment by ahadas** [comment]
Actually it depends on #334 but not really a duplicate

**Comment by Andreas0602** [comment]
Thank you, @ahadas. "find" does work now, but "save" still does not work. Is this intended? Could you make it work too?

**Comment by ahadas** [comment]
@Andreas0602 sure, [done](https://github.com/mucommander/mucommander/commit/6b85a0f1e9cfb327796afde30ea5e85577e634b0)

**Comment by Andreas0602** [comment]
Thanks again, @ahadas. You are really fast!

**Comment by ahadas** [comment]
Version 1.1.0 that includes a fix for this issue [has been released](https://github.com/mucommander/mucommander/releases/tag/1.1.0-1)
